tasks/test.py

```diff
@@ -211,6 +211,21 @@ def lint_releasenote(ctx):
 
     ctx.run("reno lint")
 
+@task
+def lint_filenames(ctx):
+    """
+    Scan files to ensure there are no filenames containing illegal character
+    """
+    forbidden_chars = '<>:"\\|?*'
```

Comment:
Does git always normalize the separators in the output to `/`? If not, we should only include `\\` if `platform.system() != 'Windows'` so this can be run on Windows.

Reply:
I guess this task need only to be run on linux, since these are all forbidden chars for windows. So there should be no risk that a windows user could commit a file with an illegal char, right ?
That said, I still need to check the platform to not run it on windows.